Update get teaching experience page and merge get teaching experience blog (#3697)
DFE-Digital/get-into-teaching-app: app/views/content/is-teaching-right-for-me/get-school-experience.md

- observe lessons
- see how teachers manage a classroom
- find out how specific subjects are taught
- speak to teachers and interact with pupils
- speak to teachers and meet pupils
- learn more about teacher training - including the application and interview process

## Find school experience

> You can search for and request [school experience](https://schoolexperience.education.gov.uk/) in England.

Reviewer: Finally, I wonder if it is helpful to give some context re the school experience service, as the blog does (but probably more briefly!). Eg you can search for schools in your area and by age group/subject. Also there are both in-person and virtual experiences on offer (although I think more in-person and you can't filter by this - but still worth mentioning?

> Once you've selected chosen a school, you can get more information about their placement availability and any requirements they may have. Some schools may charge fees to cover their costs and you may need to have a DBS check. 

Reviewer 2: This doesn't read right, not sure if chosen is there by mistake 'once you've selected a school'

- observe lessons
- see how teachers manage a classroom
- find out how specific subjects are taught
- speak to teachers and meet pupils
- learn more about teacher training - including the application and interview process

## Find school experience

> If you enter your postcode into our Get school experience service you can find schools in your area that offer placements. You can filter these by the subject and age group you're looking to teach. It also tells you whether the type of experience available is in-school, virtual or both.

Reviewer: ```suggestion
Use our Get school experience service to search for and request experience in England.

If you enter your postcode, you can find schools in your area that offer placements. You can filter these by the subject and age group you're looking to teach. It also tells you whether the type of experience available is in-school, virtual or both.
```

Would it be worth introducing the service before talking about the filters? But otherwise I think this section makes much more sense now!

Once you've chosen a school, you can get more information about their placement availability and any requirements they may have. Some schools may charge fees to cover their costs and you may need to have a DBS check. 

$get-school-experience$

### Arrange school experience independently

You can also organise school placements yourself by contacting your old school. If you’re a student you can check whether your university has any existing links with local schools. 

Or you can call a school directly. You can [get information about schools near you](https://get-information-schools.service.gov.uk/) and ask who to talk to about getting school experience so your request goes to the right person. 

Bear in mind that schools are busy places, so they may not always respond quickly and you may need to be flexible about when you visit. 

> ## Other ways to get experience of the classroom

Reviewer: ```suggestion
## Other ways to get experience
```
Should we shorten this h2 now that we have an h3 of Beyond the classroom?

### Watch pre-recorded lessons

> Many school and training provider websites have helpful information like “day in the life” videos and recorded lessons. You can also [observe teachers’ lessons](https://teachers.thenational.academy/lessons-for-itt) on the Oak National Academy website to help you get to know teaching better, before or alongside your ITT.

Reviewer: ```suggestion
Many school and training provider websites have helpful information like “day in the life” videos and recorded lessons. You can also [observe teachers’ lessons](https://teachers.thenational.academy/lessons-for-itt) on the Oak National Academy website to help you get to know teaching better, before or alongside your teacher training course.
```

We should avoid using acronyms like this - people using our site won't necessarily know what this means (I know it comes from the blog!). NB the style guide should have guidance on phrases to use.
